parquet-hadoop/src/main/java/org/apache/parquet/filter2/compat/RowGroupFilter.java

```diff
@@ -18,9 +18,7 @@
  */
 package org.apache.parquet.filter2.compat;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
```

Comment:
yeah i think intellij did this to me, i can revert this back to the old import style